tutorials/Measurements/Measurements.ipynb

```diff
@@ -0,0 +1,370 @@
+    "\n",
+    "    using (qs = Qubit[numberOfQubits])\n",
+    "    {\n",
+    "        let q0 = qs[0];\n",
+    "\n",
+    "        H (q0);\n",
+    "\n",
+    "        for (i in 1..(numberOfQubits - 1))\n",
+    "        {\n",
+    "            CNOT (q0, qs[i]);\n",
+    "        }\n",
+    "\n",
+    "        for (i in 0..(numberOfQubits - 1))\n",
+    "        {\n",
+    "            set result w/= i <- M (qs[i]);\n",
+    "        }\n",
+    "\n",
+    "        ResetAll (qs);\n",
+    "    }\n",
+    "    return result;\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "C:/snippet_.qs(17,9): error QS6008: Multiple open namespaces contain a callable with the name \"H\". Use a fully qualified name instead. Open namespaces containing a callable with that name are Microsoft.Quantum.Intrinsic, Microsoft.Quantum.Primitive.\n",
+      "C:/snippet_.qs(21,13): error QS6008: Multiple open namespaces contain a callable with the name \"CNOT\". Use a fully qualified name instead. Open namespaces containing a callable with that name are Microsoft.Quantum.Intrinsic, Microsoft.Quantum.Primitive.\n",
+      "C:/snippet_.qs(26,33): error QS6008: Multiple open namespaces contain a callable with the name \"M\". Use a fully qualified name instead. Open namespaces containing a callable with that name are Microsoft.Quantum.Intrinsic, Microsoft.Quantum.Primitive.\n",
+      "C:/snippet_.qs(29,9): error QS6008: Multiple open namespaces contain a callable with the name \"ResetAll\". Use a fully qualified name instead. Open namespaces containing a callable with that name are Microsoft.Quantum.Intrinsic, Microsoft.Quantum.Primitive.\n"
+     ]
+    }
+   ],
+   "source": [
+    "open Microsoft.Quantum.Canon;\n",
+    "open Microsoft.Quantum.Primitive;\n",
+    "\n",
+    "operation BuildBellPair () : Result[]\n",
+    "{\n",
+    "    return PutInGHZState(2);\n",
+    "}\n",
+    "\n",
+    "operation PutInGHZState (numberOfQubits : Int) : Result[]\n",
+    "{\n",
+    "    mutable result = new Result[numberOfQubits];\n",
+    "\n",
+    "    using (qs = Qubit[numberOfQubits])\n",
+    "    {\n",
+    "        let q0 = qs[0];\n",
+    "\n",
+    "        H (q0);\n",
+    "\n",
+    "        for (i in 1..(numberOfQubits - 1))\n",
+    "        {\n",
+    "            CNOT (q0, qs[i]);\n",
+    "        }\n",
+    "\n",
+    "        for (i in 0..(numberOfQubits - 1))\n",
+    "        {\n",
+    "            set result w/= i <- M (qs[i]);\n",
+    "        }\n",
+    "\n",
+    "        ResetAll (qs);\n",
+    "    }\n",
+    "\n",
+    "    return result;\n",
+    "}"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Superposition\n",
```

Comment:
We don't need to cover Superposition here - we can safely assume that the learner is familiar with it from the previous tutorials, where it is covered quite extensively

Reply:
Ohh, my bad, I was guided my Vincent to add that portion, anyways will remove it then.
Thanks for pointing it out. Thanks for all your help.